scripts/tests/test_validate_docstrings.py

```diff
@@ -146,6 +146,37 @@ def head1(self, n=5):
         """
         return self.iloc[:n]
 
+    def order(self):
+        """
+        In this case, we are illustrating the correct ordering
+        of sections in a docstring
+
+        Parameters
+        ----------
+        pat : str
+            Pattern to check for within each element.
+        case : bool, default True
+            Whether check should be done with case sensitivity.
+        na : object, default np.nan
+            Fill value for missing data.
+
+        Returns
+        -------
+        Series
+            Original series with boolean values.
+
+        Examples
```

Comment:
If this validates that order of sections is coreect, we should have all sectiona, see also and notes, and proba ly a extended summary too